setup.py

```diff
@@ -364,6 +370,12 @@ def run(self):
     # https://github.com/pandas-dev/pandas/issues/35559
     extra_compile_args.append("-Wno-error=unreachable-code")
 
+    # rdata requires system iconv library
+    os.environ["DYLD_LIBRARY_PATH"] = ""
+    rdata_includes = ["/usr/include"]
```

Comment:
Presumably these should be no on Windows

Reply:
Correct. This section is under the `if is_platform_mac()` condition.